src/Components/List/ContactImage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
let move this file into UI kit, and add image styles for small | medium | large types